UnityProject/Assets/Scripts/Lighting/LightSource.cs

```diff
@@ -133,8 +134,9 @@ void OnDrawGizmosSelected()
 		var sprite = GetComponentInChildren<SpriteRenderer>();
 		if (sprite == null)
 			return;
-		if (relatedLightSwitch == null && !isWithoutSwitch)
+		if (relatedLightSwitch == null)
```

Comment:
this doesn't fix the exceptions, according to the screenshot they are on line 144 – most likely due to `relatedLightSwitch.transform.position` if` relatedLightSwitch` is null

Reply:
I think it does. If it's null it won't pass to that section of code, cuz I fixed the check.

Reply:
ah, I think you're right... an if-else would probably more readable tho